contrib/pyln-testing/pyln/testing/utils.py

```diff
@@ -1356,11 +1356,14 @@ def __init__(self, directory: str, target: float = 90):
 
     def wait(self):
         start_time = time.time()
+        target = self.target
         with flock(self.directory):
             # We just got the lock, assume someone else just released it
             self.current_load = 100
-            while self.load() >= self.target:
+            while self.load() >= target:
                 time.sleep(1)
+                # Don't sleep forever!
+                target -= 1
 
```

Comment:
Maybe it is a coincidence but this now is worst? 

Look how many timeouts we had in the CI
https://github.com/ElementsProject/lightning/actions/runs/4499577325/jobs/7918186188?pr=6118

Reply:
maybe should be `+= 1` if we want to exist for the while?